gui/src/router.ts

```diff
@@ -425,7 +425,10 @@ router.beforeEach(async (to, from, next) => {
       });
     } else {
       try {
-        await store.dispatch("userLogin/getUserPermissionsAction");
+        await Promise.all([
```

Comment:
error handling?

Reply:
When we use **async/await**, we rarely need **.then**, because **await** handles the waiting for us. And we can use a regular **try..catch** instead of **.catch**. That’s usually (but not always) more convenient.
And we already wrapped this block of code under the **try..catch**.